MAINTAINERS.yml

```diff
@@ -0,0 +1,1136 @@
+    status: maintained
+    maintainers:
+        - galak
+    collaborators:
+        - ulfalizer
+    files:
+        - scripts/dts/
+        - dts/bindings/
+        - dts/common/
+    labels:
+        - "area: Device Tree"
+
+Disk:
+    # Provisional entry, to be checked
+    status: orphaned
+    files:
+        - subsys/disk/
+        - include/disk/
+    labels:
+        - "area: Disk Access"
+
+Display drivers:
+    status: maintained
+    maintainers:
+        - vanwinkeljan
+    collaborators:
+        - jfischer-phytec-iot
+    files:
+        - drivers/display/
+        - dts/bindings/display/
+        - include/drivers/display.h
+        - include/display/
+        - include/drivers/display.h
+        - lib/gui/
+        - subsys/fb/
+    labels:
+        - "area: Display"
+
+Documentation:
+    # Provisional entry, to be checked
+    status: orphaned
+    collaborators:
+        - carlescufi
+        - nashif
+    files:
+        - doc/
+    files-regex:
+        - \.rst$
+        - /doc/
+    labels:
+        - "area: Documentation"
+
+"Drivers: ADC":
+    # Provisional entry, to be checked
+    status: maintained
+    maintainers:
+        - anangl
+    files:
+        - drivers/adc/
+        - include/drivers/adc.h
+        - tests/drivers/adc/
+    labels:
+        - "area: ADC"
+
+"Drivers: Audio":
+    # Provisional entry, to be checked
+    status: maintained
+    maintainers:
+        - Vudentz
+    files:
+        - drivers/audio/
+        - include/audio/
+
+"Drivers: CAN":
+    status: maintained
+    maintainers:
+        - alexanderwachter
+    collaborators:
+        - henrikbrixandersen
+        - karstenkoenig
+    files:
+        - drivers/can/
+        - dts/bindings/can/
+        - include/drivers/can.h
+        - samples/drivers/CAN/
+        - tests/drivers/can/
+    labels:
+        - "area: CAN"
+
+"Drivers: Clock control":
+    status: orphaned
+    files:
+        - drivers/clock_control/
+        - dts/bindings/clock/
+        - include/drivers/clock_control.h
+        - include/dt-bindings/clock/
+        - tests/drivers/clock_control/
+    labels:
+        - "area: Clock control"
+
+"Drivers: Console":
+    # Provisional entry, to be checked
+    status: maintained
+    maintainers:
+        - pfalcon
+    files:
+        - drivers/console/
+        - include/console/
+        - include/drivers/console/
+        - subsys/console/
+    labels:
+        - "area: Console"
+
+"Drivers: Counter":
+    # Provisional entry, to be checked
+    status: maintained
+    maintainers:
+        - nordic-krch
+    files:
+        - drivers/counter/
+        - include/drivers/counter.h
+        - tests/drivers/counter/
+    labels:
+        - "area: Counter"
+
+"Drivers: Crypto":
+    # Provisional entry, to be checked
+    status: orphaned
+    files:
+        - drivers/crypto/
+        - dts/bindings/crypto/
+        - include/crypto/
+        - samples/drivers/crypto/
+    labels:
+        - "area: Crypto / RNG"
+
```

Comment:
Miss DAC

Reply:
Added DAC, checked with @martinjaeger for availability